docs/api.md

```diff
@@ -264,9 +265,72 @@
+      - [bot.deactivateItem()](#botdeactivateitem)
+      - [bot.useOn(targetEntity)](#botuseontargetentity)
+      - [bot.attack(entity)](#botattackentity)
+      - [bot.swingArm([hand], showHand)](#botswingarmhand-showhand)
+      - [bot.mount(entity)](#botmountentity)
+      - [bot.dismount()](#botdismount)
+      - [bot.moveVehicle(left,forward)](#botmovevehicleleftforward)
+      - [bot.setQuickBarSlot(slot)](#botsetquickbarslotslot)
+      - [bot.craft(recipe, count, craftingTable, [callback])](#botcraftrecipe-count-craftingtable-callback)
+      - [bot.writeBook(slot, pages, [callback])](#botwritebookslot-pages-callback)
+      - [bot.openContainer(containerBlock or containerEntity)](#botopencontainercontainerblock-or-containerentity)
+      - [bot.openChest(chestBlock or minecartchestEntity)](#botopenchestchestblock-or-minecartchestentity)
+      - [bot.openFurnace(furnaceBlock)](#botopenfurnacefurnaceblock)
+      - [bot.openDispenser(dispenserBlock)](#botopendispenserdispenserblock)
+      - [bot.openEnchantmentTable(enchantmentTableBlock)](#botopenenchantmenttableenchantmenttableblock)
+      - [bot.openAnvil(anvilBlock)](#botopenanvilanvilblock)
+      - [bot.openVillager(villagerEntity)](#botopenvillagervillagerentity)
+      - [bot.trade(villagerInstance, tradeIndex, [times], [cb])](#bottradevillagerinstance-tradeindex-times-cb)
+      - [bot.setCommandBlock(pos, command, [options])](#botsetcommandblockpos-command-options)
+      - [bot.supportFeature(name)](#botsupportfeaturename)
+      - [bot.waitForTicks(ticks)](#botwaitforticksticks)
+    - [Lower level inventory methods](#lower-level-inventory-methods)
+      - [bot.clickWindow(slot, mouseButton, mode, cb)](#botclickwindowslot-mousebutton-mode-cb)
+      - [bot.putSelectedItemRange(start, end, window, slot, noWaiting)](#botputselecteditemrangestart-end-window-slot-nowaiting)
+      - [bot.putAway(slot, noWaiting)](#botputawayslot-nowaiting)
+      - [bot.closeWindow(window)](#botclosewindowwindow)
+      - [bot.transfer(options, cb)](#bottransferoptions-cb)
+      - [bot.openBlock(block)](#botopenblockblock)
+      - [bot.openEntity(entity)](#botopenentityentity)
+      - [bot.moveSlotItem(sourceSlot, destSlot, cb)](#botmoveslotitemsourceslot-destslot-cb)
+      - [bot.updateHeldItem()](#botupdatehelditem)
+      - [bot.getEquipmentDestSlot(destination)](#botgetequipmentdestslotdestination)
+    - [bot.creative](#botcreative)
+      - [bot.creative.setInventorySlot(slot, item, [callback])](#botcreativesetinventoryslotslot-item-callback)
+      - [bot.creative.flyTo(destination, [cb])](#botcreativeflytodestination-cb)
+      - [bot.creative.startFlying()](#botcreativestartflying)
+      - [bot.creative.stopFlying()](#botcreativestopflying)
+<<<<<<< Updated upstream
       - [bot.getControlState(control)](#botgetcontrolstatecontrol-state)
       - [bot.getExplosionDamages(entity, position, radius, [rawDamages])](#botgetexplosiondamages)
+=======
+>>>>>>> Stashed changes
```

Comment:
is this intended?

Reply:
Yeah that was an accident because I'm bad at git, fixed now.